cmd/minimega/cc_cli.go

```diff
@@ -236,6 +238,85 @@ func cliCC(ns *Namespace, c *minicli.Command, resp *minicli.Response) error {
 
 // tunnel
 func cliCCTunnel(ns *Namespace, c *minicli.Command, resp *minicli.Response) error {
+	v := c.StringArgs["vm"]
+
+	if c.BoolArgs["close"] {
+		vm := ns.FindVM(v)
+		if vm == nil {
+			return vmNotFound(v)
+		}
+
+		id := c.StringArgs["id"]
+		tid, err := strconv.Atoi(id)
+		if err != nil {
+			return fmt.Errorf("invalid format for tunnel ID")
+		}
+
+		if err := ns.ccServer.CloseForward(vm.GetUUID(), tid); err != nil {
+			return err
+		}
+
+		return nil
+	}
+
+	if c.BoolArgs["list"] {
+		// map VM name --> VM UUID
+		vms := make(map[string]string)
+
+		if v == Wildcard {
+			clients := ns.ccServer.GetClients()
+
+			for _, client := range clients {
+				vms[client.Hostname] = client.UUID
```

Comment:
Small issue I noticed here. I had two VMs booted that had the same hostname, so they clobbered each other, and only 1 showed. It's also slightly inconsistent that `all` outputs hostnames, while you need the actual vm name or UUID when specifying a specific vm.

Maybe we do an extra lookup to convert from UUID -> name then use that as the key?

Reply:
My latest commits should address this as well.